### ISSUE TITLE
Words like "expression" and "overexpression" shouldn't be flagged as entity modifications

### ISSUE THREAD

**Issue by johnbachman** [clulab/reach#606]
Frequently certain generic terms like "expression" and "overexpression" are getting flagged as entity modifications in the REACH output. This is tricky for us because this ends up getting treated as a refinement on the agent in our assembly process, even though these terms don't really reflect a meaningful difference in agent state. Here are some examples:

Ras:

> Of note, oncogenic Ras expression also triggered RAD17 phosphorylation in normal diploid fibroblasts (XREF_FIG, bottom), indicating that other mitogenic oncogenes can activate RAD17.

MALT1:

> Expression of MALT1 did not induce IkappaB phosphorylation or degradation, or increase NIK or p52.

WOX1 (also note that the polarity modifier "knock-down" appears to be missed here):

>Remarkably knock-down of WOX1 expression by small interfering RNA in neuroblastoma SK-N-SH cells spontaneously induced Tau phosphorylation at Thr212 and Thr231 and Ser515 and Ser516, enhanced phosphorylation of glycogen synthase kinase 3beta (GSK-3beta) and ERK, and enhanced NFT formation.

GDF15:

>Exogenous and endogenous GDF15 over-expression stimulated phosphorylation of p38, Erk1/2, and Akt.

LCN2:

>Chung et al. recently reported that the enhanced expression of LCN2 led to the phosphorylation of PI3K-Akt and JNK in human aortic smooth muscle cells [XREF_BIBR].

ADM:

>In addition, ADM overexpression increased Akt and GSK3beta phosphorylation, and Bcl-2 and Bax ratio, and decreased the activation of caspase-3.

G3BP1:

>Overexpression of G3BP1 enhanced the phosphorylation of Smad2 and Smad3, whereas downregulation of G3BP1 attenuated the phosphorylation of Smads in the presence or absence of exogenous TGF-beta1.

ATG9A:

>Our data showed that ATG9A ectopic expression in BT474-TR induced a significant increase of c-Cbl phosphorylation at tyrosine 774 (Y774) and a moderate increase at tyrosine 731 (Y731), two known sites involved in the regulation of c-Cbl protein degradation.

More examples available if desired!

**Comment by MihaiSurdeanu** [comment]
Oh, we always thought these are important to capture... So, what would like done for these? Nothing?

**Comment by MihaiSurdeanu** [comment]
@enoriega, @maxaalexeeva: something for your attention.

**Comment by johnbachman** [comment]
@MihaiSurdeanu One option is to do nothing, especially if it's just referring to "expression" in the context of natural variation. The other option is to flag it as a modification of type "overexpression" or "amount_increase" or something like that. Part of the problem is that these are included as modifications in the REACH output but have type "UNKNOWN". If they were typed we could give them special handling during assembly.

**Comment by MihaiSurdeanu** [comment]
@zhengzhongliang: can you please run the MALT1 example, and paste here what happens? I'd like to see what the output looks like, and what rule matched the "Expression of MALT1" part of the text.

**Comment by MihaiSurdeanu** [comment]
@enoriega agreed trying to understand why this happens. Then point @zhengzhongliang to actionable things.

**Comment by enoriega** [comment]
I'm running the following sentence, simplifies from the MALT1 example provided by @johnbachman :

> Expression of MALT1 did not induce IkappaB phosphorylation

<img width="690" alt="image" src="https://user-images.githubusercontent.com/4502559/49247229-00fd0b80-f3d4-11e8-96c8-20fb7be284e8.png">

And get _four_ events: A phosphorylation, a transcription and two regulations. Both the phosphorylation and the transcription (which is triggered by `Expression` and comes from rule `transcription_1b`, which I quote at the end of this post) are straight forward. But the two regulations are redundant:

They're almost identical, but differ on their `controller`. For one, it is the transcription event: `Expression of MALT1` and for the other is the entity itself `MALT1`. 

I'm not clear on why the expression term is being flagged as an entity modification on the output, perhaps is a post processing step by the reach output code that flattens the complex event and encodes the expression event as a modification of its theme? @MihaiSurdeanu.

In any case, maybe the regulation where the entity is the controller should be filtered out given that the controller is the expression of that entity. I think the correct place for this would be an action.

Which is the specific output format being used by HMS? @johnbachman 

> - name: transcription_1b
>   priority: ${ priority }
>   example: "expression of NRF2 by Kras"
>   label: Transcription
>   pattern: |
>     trigger = [lemma=/transcription|expression/ & !outgoing=amod]
>     theme:BioChemicalEntity = nmod_of compound{,2} /conj_/*
>     cause:BioChemicalEntity? = nmod_by

**Comment by enoriega** [comment]
This is the shell output for the same sentence, where the duplication of the regulation can be appreciated more detailedly:

> EVENTS:   4
> 
> MENTION TEXT:  IkappaB phosphorylation
> LABELS:        List(Phosphorylation, AdditionEvent, SimpleEvent, Event, PossibleController)
> DISPLAY LABEL: Phosphorylation
> 	------------------------------
> 	RULE => Phosphorylation_syntax_4_noun
> 	TYPE => BioEventMention
> 	------------------------------
> 
> 	------------------------------
> 	TRIGGER => phosphorylation
> 	theme (List(Gene_or_gene_product, MacroMolecule, Equivalable, BioChemicalEntity, BioEntity, Entity, PossibleController)) => IkappaB
> CONTEXT: NONE
> 	------------------------------
> 
> 
> 
> MENTION TEXT:  Expression of MALT1 did not induce IkappaB phosphorylation
> LABELS:        List(Positive_regulation, Regulation, ComplexEvent, Event, PossibleController)
> DISPLAY LABEL: Positive_regulation
> 	------------------------------
> 	RULE => Positive_regulation_syntax_1_verb
> 	TYPE => BioEventMention
> 	------------------------------
> 
> 	with 1 modification =>
> 		Negated by "not"
> 	------------------------------
> 	TRIGGER => induce
> 	controlled (List(Phosphorylation, AdditionEvent, SimpleEvent, Event, PossibleController)) => IkappaB phosphorylation
> 	controller (List(Transcription, IncreaseAmount, Amount, SimpleEvent, Event, PossibleController)) => Expression of MALT1
> CONTEXT: NONE
> 	------------------------------
> 
> 
> 
> MENTION TEXT:  MALT1 did not induce IkappaB phosphorylation
> LABELS:        List(Positive_regulation, Regulation, ComplexEvent, Event, PossibleController)
> DISPLAY LABEL: Positive_regulation
> 	------------------------------
> 	RULE => Positive_regulation_syntax_1_verb
> 	TYPE => BioEventMention
> 	------------------------------
> 
> 	with 1 modification =>
> 		Negated by "not"
> 	------------------------------
> 	TRIGGER => induce
> 	controlled (List(Phosphorylation, AdditionEvent, SimpleEvent, Event, PossibleController)) => IkappaB phosphorylation
> 	controller (List(Gene_or_gene_product, MacroMolecule, Equivalable, BioChemicalEntity, BioEntity, Entity, PossibleController)) => MALT1
> CONTEXT: NONE
> 	------------------------------
> 
> 
> 
> MENTION TEXT:  Expression of MALT1
> LABELS:        List(Transcription, IncreaseAmount, Amount, SimpleEvent, Event, PossibleController)
> DISPLAY LABEL: Transcription
> 	------------------------------
> 	RULE => transcription_1b
> 	TYPE => BioEventMention
> 	------------------------------
> 
> 	------------------------------
> 	TRIGGER => Expression
> 	theme (List(Gene_or_gene_product, MacroMolecule, Equivalable, BioChemicalEntity, BioEntity, Entity, PossibleController)) => MALT1
> CONTEXT: NONE
> 	------------------------------

**Comment by MihaiSurdeanu** [comment]
Yeah... This happens because the controller of a regulation can be both an event and an entity.
But I do agree that there should be an action to remove the Regulation with the entity controller when a similar Regulation where the trigger is an event over the entity above.
@johnbachman: what do you think?
